scss/components/_article.scss

```diff
@@ -0,0 +1,223 @@
+.article{
+    padding: 0 20px;
+    #article-container{
+        & > div:first-child{
+            max-width: 1200px;
+            margin: 0 auto;
+            padding: 60px 0px 190px;
+            display: flex;
+            flex-direction: column;
+            @include md-screen{
+                padding: 45px 0 70px;
+            }
+            .date{
+                p {
+                    color: #383F43;
```

Comment:
Please use variables for all colors.

Reply:
@Prystal254 I'm still seeing 80 instances of hex codes being used instead of variables.